proto/eth/v2/beacon_block.proto

```diff
@@ -41,8 +41,17 @@ message BlockSSZResponseV2 {
 }
 
 message BeaconBlockContainerV2 {
-  v1.BeaconBlock phase0Block = 1;
-  BeaconBlockAltair altairBlock = 2;
+  oneof block {
+    v1.BeaconBlock phase0Block = 1;
+    BeaconBlockAltair altairBlock = 2;
+  }
+
+  // 96 byte BLS signature from the validator that produced this block.
+  bytes signature = 3 [(ethereum.eth.ext.ssz_size) = "96"];
+}
+
+message SignedBeaconBlockAltair {
+  BeaconBlockAltair message = 1;
 
   // 96 byte BLS signature from the validator that produced this block.
   bytes signature = 3 [(ethereum.eth.ext.ssz_size) = "96"];
```

Comment:
Want to change this to index 2 while you're changing this message?

Reply:
Yes, I want to :-)